data-prepper-core/src/main/java/org/opensearch/dataprepper/core/parser/model/DataPrepperConfiguration.java

```diff
@@ -76,6 +77,8 @@ public DataPrepperConfiguration(
             @JsonProperty("server_port")
             @JsonAlias("serverPort")
             final String serverPort,
+            @JsonProperty("no_peer_forwarding_keys")
+            final List<String> noPeerForwardingKeys,
```

Comment:
This logic is setup to exclude a key. I think that is not exactly the solution. It should exclude identification_keys.

So a couple of changes here:
1. Rename this to `exclude_identification_keys`
2. Change to a list of list of string.


```
exclude_identification_keys:
- ['traceId']
- ['someId1', 'someId2']
```

The peer forwarder should still forward for a processor with identificationKeys of `traceId, someOtherValue`.